### ISSUE TITLE
Rename gpu related playbooks and roles

### ISSUE THREAD

**Issue by alequint** [ibm-mas/ansible-devops#286]
Existing `install-gpu` playbook and `gpu-install` role just install NVidia operator. It does not install a GPU node in the cluster. For that reason the name of the artifacts may mess things up for users. I would replace `gpu` by `nvidia`. Update docs accordingly.

**Comment by caroazad** [comment]
@alequint Can I do this now or should I wait on the new changes for 10.0 to be merged?

**Comment by alequint** [comment]
Sorry @caroazad , did not answer your comment above... but I see you already did right? First part of the refactoring is already merged into master, so you can go ahead, though I see @durera already incorporated that in `v10refactorpt2` branch (the next take of changes happening in v10) take a look at here: https://github.com/ibm-mas/ansible-devops/commit/43c39636667cc803e18715003215083c27e0c962#diff-0430655b8c14b6a75c4622f76f813679caa584a0119956ffa208f5cbd26145b9R35 for example

**Comment by caroazad** [comment]
I see it. I also created a PR so I will go ahead and close that.

**Comment by durera** [comment]
I think we can close this as resolved by #292, the role is now named [nvidia_gpu](https://github.com/ibm-mas/ansible-devops/tree/master/ibm/mas_devops/roles/nvidia_gpu) after the operator it installs
